extensions/panache/mongodb-panache-kotlin/runtime/src/main/kotlin/io/quarkus/mongodb/panache/kotlin/reactive/ReactivePanacheMongoRepositoryBase.kt

```diff
@@ -22,48 +22,40 @@ import java.util.stream.Stream
  *
  * @param Entity The type of entity to operate on
  * @param Id The ID type of the entity
- * @see ReactivePanacheMongoRepository<Id><Entity>
+ * @see [eactivePanacheMongoRepository<Id><Entity>]
```

Comment:
```suggestion
 * @see [ReactivePanacheMongoRepository<Id><Entity>]
```

Reply:
doh.  i have that changed.  let me see if I missed a push.  *sigh*  the wife walked up at the last minute and distracted me.  :)